### PR TITLE
ign -> gz CLI Migration : gz-tools

### PR DESCRIPTION
**Pull request by methylDragon**
See: https://github.com/gazebo-tooling/release-tools/issues/739

### CODE REVIEW

**Comment by methylDragon** [comment]
The CMakeLists.txt changes are quite scuffed.

I needed to contort it because execute_process isn't expanding the $<CONFIG> generator expression...

File: etc/gz.bash_completion.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# gz bash completion
```
Comment (chapulina) [inline etc/gz.bash_completion.sh:1]: I'm forward-porting the bash completion in #95, let's get that one in first, and then we can move the file here.

Comment (chapulina) [reply etc/gz.bash_completion.sh:1]: #95 is in

Comment (methylDragon) [reply etc/gz.bash_completion.sh:18]: Ported!

Comment (methylDragon) [reply etc/gz.bash_completion.sh:18]: I left the `ign.bash_completion.sh` alone for now though

**Review by chapulina — APPROVED** [review]
Windows tick-tock was failing, I tried various things without success, they're documented on the commit history of this PR. We don't need to block on Windows tick-tock, since Windows support is very limited at the moment anyway (see https://github.com/gazebosim/gz-tools/issues/7). So let's proceed as is.

**Pull request merged**